stations.csv

```diff
@@ -3949,7 +3949,7 @@ id;name;slug;uic;uic8_sncf;latitude;longitude;parent_station_id;country;time_zon
 5244;St-Sulpice;st-sulpice;8761534;87615344;43.775073;1.680702;;FR;Europe/Paris;f;f;f;t;f;f;FRSST;SST;t;;f;8704658;f;;f;;f;;f;;f;;f;;f;;f;;f;;;;f;;f;8700528;f;;f;FRSST;f;;f;f;;;;;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv5244
 5245;St-André (Nord);st-andre-nord;8728619;87286195;50.6544842;3.0475916;;FR;Europe/Paris;f;f;f;t;f;f;FRSTA;STA;t;;f;8704531;f;;f;;f;;f;;f;;f;8704531;f;;f;;f;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv5245
 5246;Ste-Gemme;ste-gemme;8734444;87344440;46.9000000000;-0.2833330000;;FR;Europe/Paris;f;f;f;t;f;f;FRSTG;;t;;f;8704968;f;;f;;f;;f;;f;;f;8704968;f;;f;;f;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv5246
-5247;Bâle St-Jean;bale-st-jean;8718793;87187930;47.570213;7.571982;5877;CH;Europe/Zurich;f;f;f;f;f;f;FRSTJ;;t;;f;8500016;f;;f;;f;;f;8500016;t;;f;8500016;f;;f;;f;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv5247
+5247;Basel St-Jean;basel-st-jean;8718793;87187930;47.570213;7.571982;5877;CH;Europe/Zurich;f;f;f;t;f;f;FRSTJ;;t;;f;8500016;f;;f;;f;;f;8500016;t;;f;8500016;f;;f;;f;;;;f;;f;;f;;f;;f;;f;f;;;;Basilea;Bâle;Basilea;Bâle;Bazel;Basilej;;Bázel;バーゼル;바젤;Bazylea;Basileia;Базель;;;巴塞尔;urn:trainline:public:nloc:csv5247
```

Comment:
`Basel St. Johann`

Reply:
I didn’t know that the `info:fr` field could have different information from the parent station! I will do as suggested.